Releasing.md

```diff
@@ -95,9 +95,49 @@ For the body of the post, just copy this checklist and again replace all occurre
 <!-- /wp:paragraph -->
 
 <!-- wp:paragraph -->
-<p>o Mark all 4 PRs ready for review and request review from your release wrangler buddy. <br><br><strong>Note:</strong> In some release cases (like betafixes), it's likely that the PRs could have conflicts with `trunk`. In this case, do not resolve merge conflicts by merging with `trunk` as this will introduce new and unexpected changes into the release. Instead, leave the conflicts until the release is integrated into the main apps, and then resolve the conflicts when merging the PRs back to `trunk`. Optionally, a second clone of the release branch can be created to verify the CI checks.</p>
+<p>o Mark all 4 PRs ready for review and request review from the team. </p>
 <!-- /wp:paragraph -->
 
+<!-- wp:paragraph {"style":{"color":{"background":"#fcb900ba"}}} -->
+<p class="has-background" style="background-color:#fcb900ba">⚠️ In some release cases (like beta fixes), it's likely that the PRs could have conflicts with <code>trunk</code>. In this case, do not resolve merge conflicts by merging with <code>trunk</code> as this will introduce new and unexpected changes to the release. Instead, leave the conflicts until the release is integrated into the main apps, and then resolve the conflicts when merging the PRs back to <code>trunk</code>. Optionally, a second clone of the release branch can be created to verify the CI checks.</p>
+<!-- /wp:paragraph -->
```

Comment:
This change is unrelated to the goal of this PR. It's just an update to the styles and text, to specify that the reviews should be requested by the team, as we no longer have the role of release wrangler buddy.